cmd/clusterctl/client/tree/discovery.go

```diff
@@ -22,6 +22,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
```

Comment:
@fabriziopandini Let's discuss the changes in this PR and specifically in clusterctl + how we get the clusterctl tests green